phpstan-baseline.neon

```diff
@@ -528,7 +523,7 @@ parameters:
 		-
 			message: "#^Access to an undefined property object\\:\\:\\$term_id\\.$#"
 			count: 1
-			path: include/model.php
+			path: include/Models/Languages.php
```

Comment:
I proposed a way to fix this instead.

Reply:
Done in [9a346d1](https://github.com/polylang/polylang/pull/1503/commits/9a346d1fdb65dcda6297c0852d15e292a3d41860).